FueledUtils/CombineUIKit/TapAction.swift

```diff
@@ -21,77 +21,38 @@ final class TapAction<Control: ControlProtocol>: NSObject {
-	}
-
-	private func initializePublishers() {
-		self.action.isEnabledPublisher.assign(to: \.isEnabled, withoutRetaining: self)
-			.store(in: &self.cancellables)
-		self.action.isExecutingPublisher.assign(to: \.isExecuting, withoutRetaining: self)
-			.store(in: &self.cancellables)
-	}
-	#else
-	// FIXME: (Stéphane) To be retested for the next version of Swift (after 5.3)
-	// NOTE: The code is kept as it's how it be.
-
-	// The issues here seems to be related to the closure, Swift doesn't seem to like passing them around directly
-	// from one initializer to another, hence the workaround above.
-	// It doesn't like initializer the publishers directly within the initializer, so we also have to create
-	// a method that does it for us.
-	// (I have a hunch it might be tied to the number of associated types in the `ActionProtocol` protocol)
-	convenience init<Action: ActionProtocol>(_ action: Action, input: Action.Input) {
-		self.init(action) { _ in input }
-	}
-
-	init<Action: ActionProtocol>(_ action: Action, inputTransform: @escaping (Control) -> Action.Input) {
-		self.isEnabled = action.isEnabled
-		self.isExecuting = action.isExecuting
-		self.inputTransform = { inputTransform($0) }
-		self.action = AnyAction(action)
-		super.init()
-		self.action.isEnabledPublisher.assign(to: \.isEnabled, withoutRetaining: self)
-			.store(in: &self.cancellables)
-		self.action.isExecutingPublisher.assign(to: \.isExecuting, withoutRetaining: self)
-			.store(in: &self.cancellables)
-	}
-	#endif
+//	private let inputTransform: ((Control) -> Any)
+//	private let action: AnyAction
+//	private var cancellables = Set<AnyCancellable>()
+//
+//	convenience init<Action: ActionProtocol>(_ action: Action) where Action.Input == Void {
+//		self.init(action, input: ())
+//	}
+//
+//	// FIXME: (Stéphane) To be retested for the next version of Swift (after 5.3)
+//	// The issues here seems to be related to the closure, Swift doesn't seem to like passing them around directly
+//	// from one initializer to another, hence the workaround above.
+//	// It doesn't like initializer the publishers directly within the initializer, so we also have to create
+//	// a method that does it for us.
+//	// (I have a hunch it might be tied to the number of associated types in the `ActionProtocol` protocol)
+//	convenience init<Action: ActionProtocol>(_ action: Action, input: Action.Input) {
+//		self.init(action) { _ in input }
+//	}
+//
+//	init<Action: ActionProtocol>(_ action: Action, inputTransform: @escaping (Control) -> Action.Input) {
+//		self.isEnabled = action.isEnabled
+//		self.isExecuting = action.isExecuting
+//		self.inputTransform = { inputTransform($0) }
+//		self.action = AnyAction(action)
+//		super.init()
+//		self.action.isEnabledPublisher.assign(to: \.isEnabled, withoutRetaining: self)
+//			.store(in: &self.cancellables)
+//		self.action.isExecutingPublisher.assign(to: \.isExecuting, withoutRetaining: self)
+//			.store(in: &self.cancellables)
+//	}
```

Comment:
So you're just temporarily commenting out until a time when you can more properly address the issue?